site/content/en/docs/Advanced/scheduling-and-autoscaling.md

```diff
@@ -97,17 +97,30 @@ This affects the Cluster autoscaler, Allocation Scheduling, Pod Scheduling and F
 
 #### Cluster Autoscaler
 
+{{% feature publishVersion="1.30.0" %}}
+
+{{< beta title="`eviction` API" gate="SafeToEvict" >}}
+
+If a gameserver can tolerate [being evicted](https://kubernetes.io/docs/concepts/scheduling-eviction/api-eviction/#how-api-initiated-eviction-works)
+(generally in combination with setting an appropriate graceful termination period on the gameserver pod) and you
+want the Cluster Autoscaler to compact your cluster by evicting game servers when it would allow the Cluster
+Autoscaler to reduce the number of nodes in the cluster, [Controlling Disruption]({{< relref "controlling-disruption.md" >}}) describes
+how to choose the `.eviction` setting appropriate for your `GameServer` or `Fleet`.
+
+{{% /feature %}}
+{{< feature expiryVersion="1.30.0" >}}
 When using the “Packed” strategy, Agones will ensure that the Cluster Autoscaler doesn't attempt to evict and move `GameServer` `Pods` onto new Nodes during
```

Comment:
The new page no loses this note about the default behavior. I like having the section linking to the new disruption page, but I think it should still say that the default behavior is to disable the cluster autoscaler (we don't have to say how here since we link to the other page, but keeping the description of the default behavior seems important).

Reply:
Maybe keep

"When using the “Packed” strategy, by default Agones will ensure that the Cluster Autoscaler doesn’t attempt to evict and move GameServer Pods onto new Nodes during gameplay"

At the top of the section, followed by the alert & new paragraph starting with "If a gameserver can tolerate [being evicted](https://kubernetes.io/docs/concepts/scheduling-eviction/api-eviction/#how-api-initiated-eviction-works) ..."

Reply:
Fixed. I also noticed I had a rendering issue on 1.29 - the alerts don't render well unless you put them in {{< blocks, but the markdown requires it, sooo.. I mixed them. PTAL.